checkmk_extensions/agent/robotmk_collector.ps1

```diff
@@ -26,7 +23,26 @@ function Main {
     }
 }
 
+function DetermineConfigPath {
+    [OutputType([string])]
+    param (
+        [array]$ProvidedPath
```

Comment:
Let's call it `$CommandLineArgs` or so, since we don't know yet if there actually is a path